client/checkout/api/index.js

```diff
@@ -444,6 +444,45 @@ export default class WCPayAPI {
 			} );
 	}
 
+	/**
+	 * Confirm Stripe payment with fallback for rate limit error.
+	 *
+	 * @param {Object} confirmParams Confirm payment request parameters.
+	 * @param {Object|StripeElements} elements Stripe elements.
+	 * @param {string|null} paymentIntentSecret Payment intent secret used to validate payment on rate limit error
+	 *
+	 * @return {Promise} The final promise for the request to the server.
+	 */
+	async handlePaymentConfirmation(
+		confirmParams,
```

Comment:
I would put `elements` as the first parameter, and then `confirmParams` to keep it aligned with the param order of `stripe.confirmPayment` method

Reply:
Noted. I will fix it.

Reply:
Fixed.